Adding Dapr quickstart

harrykimpel opened the pull request:
# Summary

Adding some quickstart dashboards for Dapr (www.dapr.io).

<!-- DON'T DELETE THIS SECTION BELOW IF SUBMITTING A NEW QUICKSTART -->

## Pre merge checklist

<!-- THIS CHECKLIST MUST BE FULLY COMPLETE OR YOUR PR WILL NOT BE MERGED -->

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

MichelLosier commented on install/third-party/dapr/install.yml:
```diff
@@ -0,0 +1,13 @@
+id: third-party-dapr
```
Hi @harrykimpel we've been in the process of deprecating install plans in favor of data sources, and quickstarts now require a data source over an install plan.

More information here: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1

harrykimpel replied:
@MichelLosier I kept the install plan for now, let me know if you want me to remove it. I also added a data source.

MichelLosier commented on quickstarts/dapr/config.yml:
```diff
@@ -0,0 +1,27 @@
+id: 658d36f3-aa96-4e74-93da-b74eefcaabd3
```
New quickstarts will have their id generated after their first merge, lets remove this field

harrykimpel replied:
id removed

MichelLosier commented on quickstarts/dapr/config.yml:
```diff
@@ -0,0 +1,27 @@
+installPlans:
+  - third-party-daprr
+dataSourceIds:
+  - dapr
```
Looks like this data source doesn't exist yet, we'll need to create a new `dapr` data source in this PR as mentioned above.

harrykimpel replied:
added the data source

MichelLosier commented:
Hey @harrykimpel looking great! Something I missed on the first pass is that it looks like we have two dashboards in the same directory, which our validation is complaining about. Our pipeline currently expects one dashboard defined per dashboard directory, so will need to split those dashboards out into respective directories. [More on dashboards](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards)

harrykimpel commented:
> More on dashboards

Thank you @MichelLosier. I moved the dashboards into separate folders and updated the quickstarts config to refer to these two dashboards. I hope this is as expected. If not, how do others deal with multiple dashboards. I guess it is not that unusual, right?

MichelLosier commented:
Looks great @harrykimpel ! And yep this is exactly how you handle multiple dashboards thanks!

harrykimpel commented:
> Looks great @harrykimpel ! And yep this is exactly how you handle multiple dashboards thanks!

Thank you @MichelLosier! I promise to remember all of this next time ;-)
